config/routes.rb

```diff
@@ -11,5 +11,5 @@
   get "/api/v1/posters", to: "api/v1/posters#index"
   get "api/v1/posters/:id", to: "api/v1/posters#show"
   post "api/v1/posters", to: "api/v1/posters#create"
-
+  patch "api/v1/posters/:id", to: "api/v1/posters#update"
```

Comment:
I'm still unsure of the value of including the `/` in the URI for this statement, it seems unnecessary. Potential refactor down the line.